ceph-iscsi.spec

```diff
@@ -81,31 +90,69 @@ Grafana.
 %if 0%{?with_python2}
 %{__python2} setup.py build
 %else
+%if 0%{?suse_version}
+%python3_build
+%else
 %{py3_build}
 %endif
+%endif
 
 %install
 %if 0%{?with_python2}
 %{__python2} setup.py install -O1 --skip-build --root %{buildroot}  --install-scripts %{_bindir}
 %else
+%if 0%{?suse_version}
+%python3_install
+%python_expand %fdupes %{buildroot}%{$python_sitelib}
+%else
 %{py3_install}
 %endif
+%endif
 mkdir -p %{buildroot}%{_unitdir}
 install -m 0644 .%{_unitdir}/rbd-target-gw.service %{buildroot}%{_unitdir}
 install -m 0644 .%{_unitdir}/rbd-target-api.service %{buildroot}%{_unitdir}
-mkdir -p %{buildroot}%{_sysconfdir}/systemd/system/rbd-target-gw.service.d
-install -m 0644 .%{_sysconfdir}/systemd/system/rbd-target-gw.service.d/dependencies.conf %{buildroot}%{_sysconfdir}/systemd/system/rbd-target-gw.service.d/
 mkdir -p %{buildroot}%{_mandir}/man8
 install -m 0644 gwcli.8 %{buildroot}%{_mandir}/man8/
 gzip %{buildroot}%{_mandir}/man8/gwcli.8
+%if 0%{?suse_version}
+mkdir -p %{buildroot}%{_unitdir}/rbd-target-gw.service.d
+install -m 0644 .%{_sysconfdir}/systemd/system/rbd-target-gw.service.d/dependencies.conf %{buildroot}%{_unitdir}/rbd-target-gw.service.d/
+mkdir -p %{buildroot}%{_sbindir}
+ln -s service %{buildroot}%{_sbindir}/rcrbd-target-gw
+ln -s service %{buildroot}%{_sbindir}/rcrbd-target-api
+%else
+mkdir -p %{buildroot}%{_sysconfdir}/systemd/system/rbd-target-gw.service.d
```

Comment:
Oh, yes this is a general bug in our systemd packaging here. We should install to `%{_unitdir}` on all platforms and not write files into `/etc/systemd`. Are you ok making this change for everything?

We'll still need the `suse_version` conditional here (and below in `%files`) for the rc* scripts.

Reply:
Sure, I'll fix this.